src/o365/spo/commands/feature/feature-list.ts

```diff
@@ -0,0 +1,175 @@
+import auth from '../../SpoAuth';
+import { Auth } from '../../../../Auth';
+import config from '../../../../config';
+import * as request from 'request-promise-native';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import { ContextInfo } from '../../spo';
+import Utils from '../../../../Utils';
+import { Feature } from './feature';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  url: string;
+  scope?: string;
+}
+
+class SpoFeatureListCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.FEATURE_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Lists features for site or site collection';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.scope = args.options.scope || 'Web';
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const scope: string = (args.options.scope) ? args.options.scope : 'Web';
+    const resource: string = Auth.getResourceFromUrl(args.options.url);
+    let siteAccessToken: string = '';
+
+    auth.getAccessToken(resource, auth.service.refreshToken as string, cmd, this.debug)
+      .then((accessToken: string): request.RequestPromise => {
+        siteAccessToken = accessToken;
+
+        if (this.debug) {
+          cmd.log(`Retrieved access token ${accessToken}. Retrieving request digest...`);
+        }
+
+        return this.getRequestDigestForSite(args.options.url, accessToken, cmd, this.debug);
```

Comment:
Retrieving activated features is a GET request, so we don't need request digest for it